networking/synthetic/README.md

```diff
@@ -78,6 +78,9 @@ $  ./onload_install
 ```
 $ python stac-prepare-nodes.py -s https://<api-server-ip>:<port> -n <node-1> <node-2> ... <node-N> -i eth0
 ```
+#### NOTE
+make sure to reboot your hosts.
```

Comment:
it is, but otherwise it wont work.
a better solution for that will be to run that script from 3rd machine (like controller or something) and have an ansible role to reboot the machien and make sure they come up.


meanwhile will fix the indentation thing